server/channels/web/web.go

```diff
@@ -65,7 +65,7 @@ func Handle404(a app.AppIface, w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 		w.WriteHeader(err.StatusCode)
 		err.DetailedError = "There doesn't appear to be an api call for the url='" + r.URL.Path + "'.  Typo? are you missing a team_id or user_id as part of the url?"
```

Comment:
The `DetailedError` is already overwritten here.

Reply:
I'm not sure what you mean. Then `ToJSON` will append the wrapped error. All of which shouldn't be sent if it's not developer mode.

Reply:
Yes thanks. In this case, it looks like the error is being formed from L61. Do you still think it's necessary to make any changes here?

Reply:
No need to change make changes here I think, good point.